### PR TITLE
feat(blog): establish blog guidelines

### LINKED ISSUE #1306: [Suggestion] Blog Guidelines
## Description: 

https://tag-security.cncf.io/blog/ provides a foundation for potential community collaboration and greater extension of serving information to consumers of the work that the Security TAG provides. 

This issue will comprise initial guidance for submission of blog content - encouraging participation within the TAG as well as working groups and others entities contributing to efforts. 

## Scope:

Minimal amount of effort to produce an artifact for initial state

### PR DESCRIPTION
**Pull request by brandtkeller**
Resolves #1306 

Establishes guidance around blog submission and includes a blog submission issue template.

Given that this is both good for publication on the website itself - it tried to locate a good place. `Community` felt out of place and `governance` is still a bit mixed but still applicable? open to moving it around.

Open to all feedback.

### CODE REVIEW

**Comment by brandtkeller** [comment]
Expecting the `links` check to fail currently - as the link to the submission template won't exist on main until merge

**Comment by eddie-knight** [comment]
I like it. 

Left some comments, and I noticed there is a wonky commit including marco's style PR.

File: governance/blog-guidelines.md
Comment (eddie-knight) [inline governance/blog-guidelines.md:53]: Should we add some more detail here regarding who is reviewing and who is approving/merging?

File: governance/blog-guidelines.md
Comment (mrcdb) [reply governance/blog-guidelines.md:41]: +1 . also IMHO we may need to clarify whether there is a naming convention for blog post markdown files and for the headings of blog posts so that they are displayed correctly in the website, e.g::

```markdown
---
title:  "Website Re-Launch at CNSC '24"
date:   2024-06-26 09:00:00 -0700
author: Eddie Knight 
---
```

File: governance/blog-guidelines.md
Comment (brandtkeller) [reply governance/blog-guidelines.md:53]: Should review be acceptable for any "maintainer" in the repository? I'm assuming that current maintainers are chairs/TL's?

File: governance/blog-guidelines.md
Comment (brandtkeller) [reply governance/blog-guidelines.md]: @mrcdb Added a section for `Front Matter` for article title. Appended an item to `Formatting` to include file naming conventions.

@eddie-knight Appended an item to Draft which contains the path to the blog article location.

File: governance/blog-guidelines.md
Comment (brandtkeller) [reply governance/blog-guidelines.md:70]: Added a `Review Criteria` Section to take a first pass at criteria.

File: governance/blog-guidelines.md
Comment (eddie-knight) [inline governance/blog-guidelines.md:75]:
```suggestion
2. **Sponsor**: Submission is sponsored by a STAG Co-Chair, Tech Lead, or Project Lead
```

File: governance/blog-guidelines.md
Comment (eddie-knight) [inline governance/blog-guidelines.md:63]:
```suggestion
2. **Submission Pull Request**: Submit your draft using the [blog submission template](/.github/ISSUE_TEMPLATE/blog-submission.md).
```

This should fix the broken link check. Though I'm not sure whether or not it needs the leading `/`

File: governance/blog-guidelines.md
Comment (eddie-knight) [reply governance/blog-guidelines.md]: Yeah, I think it'll be fine

File: governance/blog-guidelines.md
Comment (brandtkeller) [reply governance/blog-guidelines.md:63]: This is true - although it might haunt us on the website

Comment (eddie-knight) [reply governance/blog-guidelines.md:63]: Looks like it worked!

Comment (brandtkeller) [reply governance/blog-guidelines.md:63]: Open to making this change - need to think more about links that point to locations not included in the website

Comment (eddie-knight) [inline governance/blog-guidelines.md:63]: New comment on this line... I was just going back through the file from top-to-bottom on the website and noticed that this line stands out as unclear.

"Submission Pull Request" doesn't have clear meaning to me. It's even foggier when noting that the link is to a GitHub Issue Template— nothing related to a pull request.

Comment (eddie-knight) [reply governance/blog-guidelines.md:63]: Heh. I need to stop reviewing PRs in middle of long travel legs— thought you already applied the change here. If it doesn't work, we can just merge it with the hard path.

Comment (brandtkeller) [reply governance/blog-guidelines.md:63]: ah good point - I mentally combined the "submission" of the issue with the "submission" of the PR. Will revise.

**Review by eddie-knight — APPROVED** [review]
Thanks for applying the suggestions ✊ LGTM

**Pull request merged**
